src/legacy/core_plugins/embeddable_api/public/context_menu_actions/open_context_menu.tsx

```diff
@@ -0,0 +1,164 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+
+import { EuiContextMenu, EuiContextMenuPanelDescriptor, EuiPopover } from '@elastic/eui';
+import { EventEmitter } from 'events';
+import ReactDOM from 'react-dom';
+import { I18nContext } from 'ui/i18n';
+
+let activeSession: ContextMenuSession | null = null;
+
+const CONTAINER_ID = 'contextMenu-container';
```

Comment:
Subject?

Reply:
This change would cause some issues with functional tests which rely on this being a specific string.  Not opposed to the change but would prefer to punt on it in this PR specifically.